packages/utils/path.ts

```diff
@@ -139,3 +139,7 @@ export function extractExecutablePath(cmd: string) {
 
 	return output;
 }
+
+export function convertWindowsPathToPosix(path: string) {
+	return path.split('\\').join('/');
+}
```

Comment:
I see that `toForwardSlashes` is already defined in `path.ts`. Consider using `toForwardSlashes` instead of `convertWindowsPathToPosix`.

Reply:
Aside from that, this pull request looks good to me!